src/Service.Tests/ModuleInitializer.cs

```diff
@@ -25,6 +25,12 @@ public static void Init()
         VerifierSettings.IgnoreMember<DataSource>(dataSource => dataSource.ConnectionString);
         // Ignore the JSON schema path as that's unimportant from a test standpoint.
         VerifierSettings.IgnoreMember<RuntimeConfig>(config => config.Schema);
+        // Ignore the datasource files as that's unimportant from a test standpoint.
+        VerifierSettings.IgnoreMember<RuntimeConfig>(config => config.DataSourceFiles);
+        // Ignore the CosmosEngineNeeded files as that's unimportant from a test standpoint.
+        VerifierSettings.IgnoreMember<RuntimeConfig>(config => config.SqlDataSourceUsed);
+        // Ignore the SqlEngineNeeded files as that's unimportant from a test standpoint.
+        VerifierSettings.IgnoreMember<RuntimeConfig>(config => config.CosmosDataSourceUsed);
```

Comment:
I think this is indication of an undesirable behavior. `SqlDataSourceUsed` and `CosmosDataSourceUsed` should not be written to the config file. when generated by the CLI. You shouldn't have to add ignoremember because those members shouldnt be in the config file anyway

Reply:
Discussed this offline.

Reply:
<img width="953" alt="image" src="https://github.com/Azure/data-api-builder/assets/124841904/0a49f0e5-a66f-4c35-a49b-67f6b1427fb0">
See above where the cosmosDataSourceUsed and SqlDataSourceUsed do not get serialized when tojson call is made.